src/components/ChapterRow.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+  import { isOpen, chapterToDisplay } from "../store/chapterPreviewStore";
+
+  export let chapter = {};
+  export let index = 1;
+  export let description =
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur ac orci ultricies, viverra tellus sit amet, lobortis elit. Aliquam erat volutpat. Sed metus turpis, rutrum eu varius non, bibendum quis libero. Invulputate.";
+
+  function updateStateForModal() {
+    document.querySelector(".chapter-preview-container").style.display = "";
```

Comment:
Didn't really know of a better way to do this. 

The reason I have to do this is because... when someone closes the modal, I want to complete HIDE the modal. 

The problem now is... when you open it, (i.e. `:isOpen => true`), the damn HTML element still has `style="display: none;"`. 

Oh well.